.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -65,20 +67,22 @@ Branch name: `bugfix_<Issue Number>_main_<Version>_<Description>`
 - [ ] Add/update log messages for easier debugging.
 - [ ] Add/update unit tests.
 - [ ] Add/update documentation.
-- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
+- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/appendixA.html#metplus-components-python-packages) table.
+- [ ] For any new datasets, an entry to the [METplus Verification Datasets Guide](https://metplus.readthedocs.io/en/latest/Verification_Datasets/index.html).
```

Comment:
This may not be necessary for a bugfix issue since a fix to existing functionality is unlikely to add new datasets.
```suggestion
```

Reply:
@georgemccabe I thought the same thing at first, but then I thought there could be a case where the bug would have been discovered due to a new dataset.  Please confirm if you'd prefer I take this out or leave it in.  Thanks!

Reply:
I think the new dataset would be added with a different issue. If a bug was discovered, a new issue would be created to address the bug that is separate.

Reply:
Given your comment, it sounds like you would prefer that I remove it, so I will do that.  Please let me know if I misunderstood.